### PR TITLE
Added migration guide about the the queryDirection changes

### PR DESCRIPTION
**Pull request by MacondoExpress**
Described changes on the `@relationship.queryDirection`

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-graphql-208.surge.sh

Updated pages:
- [Migration to 6.0.0](https://neo4j-docs-graphql-208.surge.sh/graphql/6/migration)
- [Relationships](https://neo4j-docs-graphql-208.surge.sh/graphql/6/types/relationships)

File: modules/ROOT/pages/migration/index.adoc [hunk 1]
```diff
@@ -475,3 +475,14 @@ The `directed` argument was used to change the query direction of the relationsh
 
 The `directed` argument in `@relationship` fields is deprecated and will be removed in the future.
 Configure the query direction via the `queryDirection` and `direction` arguments of the `@relationship` directive instead.
+
+=== Deprecated `@relationship.queryDirection` `_DEFAULT` and `_ONLY` options.
```
Comment (rsill-neo4j) [inline modules/ROOT/pages/migration/index.adoc]:
```suggestion
=== Deprecated `@relationship.queryDirection` `_DEFAULT` and `_ONLY` options
```

File: modules/ROOT/pages/migration/index.adoc [hunk 2]
```diff
@@ -475,3 +475,14 @@ The `directed` argument was used to change the query direction of the relationsh
...
+
+Following the deprecation of the `directed` argument, the values accepted by the `@relationship.queryDirection` have changed:
+
+- `DEFAULT_DIRECTED` is deprecated.
+- `DEFAULT_UNDIRECTED` is deprecated.
+- `DIRECTED_ONLY` is renamed to `DIRECTED` 
+- `UNDIRECTED_ONLY` is renamed to `UNDIRECTED`
```
Comment (rsill-neo4j) [inline modules/ROOT/pages/migration/index.adoc]:
```suggestion
- `DIRECTED_ONLY` is renamed to `DIRECTED`.
- `UNDIRECTED_ONLY` is renamed to `UNDIRECTED`.
```

File: modules/ROOT/pages/migration/index.adoc [hunk 3]
```diff
@@ -475,3 +475,14 @@ The `directed` argument was used to change the query direction of the relationsh
...
+
+Following this change, a warning is raised if a deprecated value is used.
```
Comment (rsill-neo4j) [inline modules/ROOT/pages/migration/index.adoc]:
```suggestion
A warning is raised if a you use a deprecated value.
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-graphql-208.surge.sh

Updated pages:
- [Migration to 6.0.0](https://neo4j-docs-graphql-208.surge.sh/graphql/6/migration)
- [Relationships](https://neo4j-docs-graphql-208.surge.sh/graphql/6/types/relationships)

File: modules/ROOT/pages/migration/index.adoc [hunk 4]
```diff
@@ -525,3 +525,14 @@ The `directed` argument was used to change the query direction of the relationsh
...
+=== Deprecated `@relationship.queryDirection` `_DEFAULT` and `_ONLY` options
+
+Following the deprecation of the `directed` argument, the values accepted by the `@relationship.queryDirection` have changed:
+
+- `DEFAULT_DIRECTED` is deprecated.
+- `DEFAULT_UNDIRECTED` is deprecated.
+- `DIRECTED_ONLY` is renamed to `DIRECTED`.
+- `UNDIRECTED_ONLY` is renamed to `UNDIRECTED`.
+
+A warning is raised if a you use a deprecated value.
```
Comment (rsill-neo4j) [inline modules/ROOT/pages/migration/index.adoc]:
```suggestion
A warning is raised if you use a deprecated value.
```

**Review by rsill-neo4j — APPROVED** [review]
one small suggestion + approval :)

**Comment by MacondoExpress** [comment]
> one small suggestion + approval :)

Done! thank you!

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-graphql-208.surge.sh

Updated pages:
- [Migration to 6.0.0](https://neo4j-docs-graphql-208.surge.sh/graphql/6/migration)
- [Relationships](https://neo4j-docs-graphql-208.surge.sh/graphql/6/types/relationships)

**Comment by neo-technology-commit-status-publisher** [comment]
Thanks for the documentation updates.

The preview documentation has now been torn down - reopening this PR will republish it.

**Pull request merged**